ros2topic/ros2topic/verb/echo.py

```diff
@@ -128,7 +128,7 @@ def subscriber(
             raise RuntimeError(
                 'Could not determine the type for the passed topic')
 
-    msg_module = import_message_type(topic_name, message_type)
+    msg_module = get_message(message_type)
```

Comment:
@jacobperron meta^2: I wonder what'd happen if you used `nargs='?', type=get_message` above.

Reply:
I guess it would be fine if we change this line to `msg_module = message_type`. But this seems out of scope of this PR. If you'd like to change it in a follow-up, go for it :)